drake/examples/HolonomicDrive/oneWheelGradients.m

```diff
@@ -0,0 +1,43 @@
+% Autogenerated file, from makeGrads.m
```

Comment:
Yep, [we do](https://github.com/RobotLocomotion/drake/blob/master/drake/examples/Acrobot/dynamicsGradients.m#L2)